firedrake/functionspaceimpl.py

```diff
@@ -391,15 +391,15 @@ def __mul__(self, other):
 
     @property
     def node_count(self):
-        """The number of global nodes in the function space.  If the
+        """The number of local nodes in the function space.  If the
```

Comment:
How about "The number of nodes of this function space _visible_ on this process."